.circleci/config.yml

```diff
@@ -69,6 +72,14 @@ jobs:
       - image: circleci/node:12
     steps:
       - checkout
+      - run:
+          name: Sync with latest master branch (only on PRs)
+          command: |
+            if [ -n "$CIRCLE_PULL_REQUEST" ]
```

Comment:
Is it possible to use [`branches`](https://circleci.com/docs/2.0/configuration-reference/#branches-2) or `filters` instead?

Reply:
I don't know if we can configure them on dynamic refspecs though?

Reply:
I only see this sparse doc on this: https://circleci.com/docs/2.0/workflows/#using-regular-expressions-to-filter-tags-and-branches

Reply:
Oh ok then, it's probably not worth investigating for a single line.

Reply:
that was also motivator for this PR: https://github.com/babel/babel-test262-runner/pull/16